build_library/release_util.sh

```diff
@@ -42,6 +40,43 @@ DEFINE_string sign "" \
   "Sign all files to be uploaded with the given GPG key."
 DEFINE_string sign_digests "" \
   "Sign image DIGESTS files with the given GPG key."
+DEFINE_string image_compression_formats "${DEFAULT_IMAGE_COMPRESSION_FORMAT}" \
+  "Compress the resulting images using thise formats. This option acceps a list of comma separated values. Options are: none, bz2, gz, zip, zstd"
+
+
+compress_file() {
+    local filepath="$1"
+    local compression_format="$2"
+
+    [ ! -f "${filepath}" ] && die "Image file ${filepath} does not exist"
+    [ -z "${compression_format}" ] && die "compression format parameter is mandatory"
+
+    case "${compression_format}" in
+    "none"|"")
+        echo -n "${filepath}"
+        return 0
+        ;;
+    "bz2")
+        IMAGE_ZIPPER="lbzip2 --compress --keep"
+        ;;
+    "gz")
+        IMAGE_ZIPPER="pigz --keep"
+        ;;
+    "zip")
+        IMAGE_ZIPPER="pigz --keep --zip"
+        ;;
+    "zstd")
+       IMAGE_ZIPPER="zstd --format=zstd -k -q -f --no-progress -o ${filepath}.${compression_format}"
```

Comment:
```suggestion
       IMAGE_ZIPPER="zstd --format=zstd -k -q --no-progress -o ${filepath}.${compression_format}"
```
I see that `-f` is specified below. (Any of the two places is fine but it should be consitent.)

Reply:
Oops. Missed that one. Fixed.